src/main/java/Duke.java

```diff
@@ -1,10 +1,283 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.*;
+
 public class Duke {
+    private boolean isActive;
+    private List<Task> myList;
```

Comment:
This might be better if it was `tasks` as its more descriptive of what the list is